editor/src/components/navigator/left-pane.tsx

```diff
@@ -657,6 +660,28 @@ const SharingPane = betterReactMemo('SharingPane', () => {
 })
 
 const GithubPane = betterReactMemo('GithubPane', () => {
+  const [githubRepoStr, setGithubRepoStr] = React.useState('')
+  const parsedRepo = parseGithubProjectString(githubRepoStr)
```

Comment:
Does this need memoisation?

Reply:
Nah, because it will only be called whenever the `githubRepoStr` changes